pages/linux/hd-idle.md

```diff
@@ -0,0 +1,16 @@
+# hd-idle
+
+> Utility program for spinning-down external disks after a period of idle time.
+> More information: <https://hd-idle.sourceforge.net/>.
```

Comment:
```suggestion
> More information: <https://manned.org/hd-idle>.
```
There's no anchor point in the official documentation webpage to automatically scroll to the usage section. I'd use manned in this case.